airbyte-server/src/main/resources/micronaut-banner.txt

```diff
@@ -6,3 +6,6 @@
 /_/  |_/_/_/  /_.___/\__, /\__/\___/
                     /____/
         : airbyte-server :
+--------------------------------------------------
+ Will soon be available at http://localhost:8000/
+--------------------------------------------------
```

Comment:
Should we leave this message?

Reply:
I have added it on purpose because the old server is available is not displayed anymore